config/routes.rb

```diff
@@ -14,4 +14,6 @@
   get '/login', to: 'sessions#new'
 
   post '/login', to: 'sessions#create', as: 'sessions'
+
+  delete '/logout', to: 'sessions#destroy'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.